src/serialize.h

```diff
@@ -211,7 +211,7 @@ template<typename X> const X& ReadWriteAsHelper(const X& x) { return x; }
  *
  * Example use:
  *   struct FooFormatter {
- *     FORMATTER_METHODS(Class, obj) { READWRITE(obj.val1, VARINT(obj.val2)); }
+ *     FORMATTER_METHODS(Class, obj) { READWRITE(obj.val1, VARINT(obj.val2, VarIntMode::DEFAULT)); }
```

Comment:
:eyes:

Reply:
Aha, someone caught it!

(*clearly* I left that in to see if reviewers were paying attention)

Reply:
Fixed.